pallets/committee/src/lib.rs

```diff
@@ -468,6 +468,7 @@ impl<T: Config<I>, I: Instance> Module<T, I> {
             return;
         }
 
+        // TODO: Either remove or use a fixed DID?
         let did = Context::current_identity::<Identity<T>>().unwrap_or_default();
         Self::finalize_proposal(approved, seats, ayes, nays, proposal, did);
         let event = RawEvent::FinalVotes(did, voting.index, proposal, voting.ayes, voting.nays);
```

Comment:
The `execute_if_passed` function here doesn't always have access to caller details.  It is called in two places and one is code that we can't provide caller details too (Trait impl).

Reply:
Let's check w/ the integration team how useful it is to keep a DID here - if it isn't useful for SQ / SDK we can just remove it.

Reply:
Thinking out loud only. Even if we define our own `ChangeMembers` trait we wouldn't be able to pass the caller did?

Reply:
There are only about 4 or 5 calls to the `change_members_sorted` method of that `ChangeMembers` trait, and it looks like we would have at least the caller's AccountId.  Also we only use it in our `group` pallet, we don't use any of the Substrate pallets that uses `ChangeMembers`.  So we don't even need to modify the Substrate trait, we can just define our own.

Reply:
We can make this DID optional in the event.  Since there is at least one call that can provide a DID.